New example folder (#720)
flow-project/flow: examples/README.md

# Flow Examples

Before continuing to the Flow examples, we recommend **installing Flow** by 
following the [installation instructions](
https://flow.readthedocs.io/en/latest/flow_setup.html).

The **examples** folder provides several examples demonstrating how 
both non-RL simulation and RL-oriented simulatons can be setup and executed 
within the Flow framework on a variety of traffic problems. These examples are 
python files that may be executed either from terminal or via a text editor (or 

> Python IDE). For example, in order to execute the non-RL Ring example we run:

Author: can you run this out of IDE? since you have to pass arguments now that might not be true

Reviewer: I resolved this in my commit (removed "Python IDE") @AboudyKreidieh

To execute these examples, run

```shell
python simulate.py EXP_CONFIG 
```
where `EXP_CONFIG` is the name of the experiment configuration file, as located in 
`exp_configs/non_rl.`

There are several *optional* arguments that can be added to the above command:

```shell
 python simulate.py EXP_CONFIG --num_runs n --render --aimsun --gen_emission
```
where `--num_runs` indicates the number of simulations to run (default of `n` is 1), `--render` indicates whether to run the simulation during runtime (default is False), `--aimsun` indicates whether to run the simulation using the simulator Aimsun (the default simulator is SUMO), and `--gen_emission` indicates whether to generate an emission file from the simulation.

**RL examples based on RLlib** provides similar networks as those presented in 
the first point, but in the present of autonomous vehicle (AV) or traffic light agents 
being trained through RL algorithms provided by *RLlib*.

To execute these examples, run

```shell
 python train_rllib.py EXP_CONFIG
```
where `EXP_CONFIG` is the name of the experiment configuration file, as located in 
`exp_configs/rl/singleagent` or  `exp_configs/rl/multiagent.`

> **RL examples based on "stable" baselines** provides similar networks as those 

Author: ```suggestion
**RL examples based on stable_baselines** provides similar networks as those 
```

![](https://raw.githubusercontent.com/flow-project/flow/master/docs/img/highway.gif)

### highway_ramps.py

Example of a highway section network with on/off ramps

> ![](picture to be added)

Reviewer 2: Can you make sure there are issues / other PRs for these TODOs

Reviewer: Yeah sure. I am making new PR for most of the TODOs, like more important stuff such as tests and benchmarks.

### merge.py

> Example of a straight road with merge-in with human-driven vehicles.

Author: ```suggestion
Example of a straight road with an on-ramp merge.
```
